src/main/java/org/prebid/server/settings/proto/response/HttpAccountsResponse.java

```diff
@@ -0,0 +1,14 @@
+package org.prebid.server.settings.proto.response;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+import java.util.Map;
+
+@AllArgsConstructor(staticName = "of")
+@Value
+public class HttpAccountsResponse {
+
+    Map<String, ObjectNode> accounts;
```

Comment:
Since account payload is mapped to `Account` in `HttpApplicationSettings` it makes sense to do this conversion one time when `HttpAccountResponse` is constructed out of response body by changing this field to have `Map<String, Account>` type.